src/App.js

```diff
@@ -52,11 +51,11 @@ function App() {
             <Route path="/users/login">
               <LoginForm setSessionToken={setSessionToken} />
             </Route>
-            <Route path="/users">
+            <Route path="/users/:id">
               <UserContainer sessionToken={cookies.sessionToken} />
             </Route>
             <Route path="/:id">
-              <ConferenceContainer />
+              <ConferenceContainer sessionToken={cookies.sessionToken} />
```

Comment:
I don't think you need to pass the sessionToken prop

Reply:
Did you mean to keep the sessionToken?

Reply:
oops sorry I forgot to remove this for the other components too!